databricks/koalas/mlflow.py

```diff
@@ -66,7 +66,8 @@ def _model(self) -> Any:
     @lazy_property
     def _model_udf(self):
         spark = default_session()
-        return spark_udf(spark, self._path, self._run_id, result_type=self._return_type)
+        model_uri = "runs:/{run_id}/{path}".format(run_id=self._run_id, path=self._path)
```

Comment:
Good question! Actually the uri should come from the user. IT does not have to be runs uri, it can point to e.g. se3, hdfs or any other supported uri type.